app/views/components/question.scala.html

```diff
@@ -20,10 +20,19 @@
 
 @(q: Question, name: String, form: Form[_])(implicit request: Request[_], messages: Messages)
 
+@ariaDescribedBy(s: String) = { aria-describedby="@s" }
+
+@optionalAria(a: Question) = @{
+  (q.hint.map(_ => "question-hint")::q.errorMsgs.headOption.map(err => s"${err.id}-error")::Nil).flatten match {
```

Comment:
That is an inspired line. If comments worked I would suggest you add one to explain what is going on!

Reply:
;)